gensim/test/test_nmf.py

```diff
@@ -87,18 +87,18 @@ def testTransform(self):
         transformed = self.model[doc]
 
         vec = matutils.sparse2full(transformed, 2)  # convert to dense vector, for easier equality tests
-        expected = [0., 1.]
+        expected = [0.02991635, 0.97008365]
         # must contain the same values, up to re-ordering
-        self.assertTrue(np.allclose(sorted(vec), sorted(expected)))
+        self.assertTrue(np.allclose(sorted(vec), sorted(expected), rtol=1e-2))
 
         # transform one word
         word = 5
         transformed = self.model.get_term_topics(word)
 
         vec = matutils.sparse2full(transformed, 2)
-        expected = [0.35023746, 0.64976251]
+        expected = [[0.3076869, 0.69231313]]
         # must contain the same values, up to re-ordering
-        self.assertTrue(np.allclose(sorted(vec), sorted(expected), rtol=1e-4))
+        self.assertTrue(np.allclose(sorted(vec), sorted(expected), rtol=1e-2))
```

Comment:
Why have we increased the tolerance? 1e-2 seems rather high. What has changed to justify this?

Reply:
The same reason as in the comment above.

Reply:
But the comment above talks about Windows, and I still see this 1e-2 (not 1e-4) even on non-Windows.

On Windows, it's even more lax 1e-1 (aka more-or-less random results).